frontend/src/index.css

```
@@ -12,7 +15,7 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-@import "./themes/default.css";
+@import "./themes/base.css";
```

Stylelint (17.9.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)